TrackPropagation/RungeKutta/test/rkTest_cfg.py

```diff
@@ -25,10 +25,12 @@
 
 #process.load("Configuration.StandardSequences.MagneticField_35T_cff")
 
-process.load("Configuration.StandardSequences.MagneticField_38T_cff")
+#process.load("Configuration.StandardSequences.MagneticField_38T_cff")
 
 #process.load("Configuration.StandardSequences.MagneticField_40T_cff")
 
+process.load("MagneticField.Engine.volumeBasedMagneticField_090322_2pi_scaled_cfi")
```

Comment:
I need to access a specific magnetic field map, w/o events, w/o a run, e/o database.
this is a clear use case and shall be supported!
Tell me how I can access a 3.8T magnetic field corresponding to a given "era" in a unit-test please.
just provide a working example